### DISCUSSION TITLE
Configure redux-persist with redux-toolkit

### DISCUSSION THREAD

**Discussion by somratAtSekaiLab** [monstar-lab-oss/admin-panel-template-reactjs#61] [Ideas]
How to configure redux-persist with redux-toolkit, its best practices, pros and cons. 
Please share your thoughts with code example.

Findings: 
https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist

**Comment by yukinoda** [comment]
@somratAtSekaiLab cc: @mvn-binhnguyen-hn

> https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist

In a different project, we are using totally like this. 
And whitelisting the states to persisting them to localStorage instead of manually doing with  `localStorage.getItem` or `localStorage.setItem`

FYI:
found an old discussion about [Why use redux-persist over manually persisting state to localStorage?](https://stackoverflow.com/questions/49222396/why-use-redux-persist-over-manually-persisting-state-to-localstorage)
